tests/app/authentication/test_authentication.py

```diff
@@ -97,12 +97,12 @@ def test_auth_should_not_allow_request_with_invalid_iat(client, sample_api_key,
         'iat': 'not an integer'
     }
 
-    token = jwt.encode(payload=claims, key=str(uuid.uuid4()), headers=headers).decode()
+    token = jwt.encode(payload=claims, key=str(uuid.uuid4()), headers=headers)
 
     request.headers = {'Authorization': f'Bearer {token}'}
     with pytest.raises(AuthError) as exc:
         requires_auth()
-    assert exc.value.short_message == 'Invalid token: Issued At claim (iat) must be an integer.'
+    assert exc.value.short_message.startswith('Invalid token:')
```

Comment:
Upgrading the version made this message disappear because the Python disables signature verification and this code disables iat check https://github.com/jpadilla/pyjwt/commit/373d4d8a45ce7071e8677d4835a46bd762fa1210 and https://github.com/alphagov/notifications-python-client/blob/cf7216492fae3a9fb4ee1962ce940f856cfac0d8/notifications_python_client/authentication.py#L145